pkg/koordlet/metricsadvisor/collectors/coldmemoryresource/cold_page_kidled_test.go

```diff
@@ -530,8 +530,9 @@ func Test_collectNodeColdPageInfo(t *testing.T) {
 	}
 	assert.NoError(t, err)
 	got1, got2 := testGetNodeMetrics(t, c.metricDB, testNow, 5*time.Second)
-	assert.Equal(t, float64(7.33569024e+08), got1)
-	assert.Equal(t, float64(3.401728e+08), got2)
+	// memInfo.MemTotal*1024 - memInfo.MemAvailable*1024 + memInfo.ActiveFile*1024 + memInfo.InactiveFile*1024 - coldPageUsageBytes
```

Comment:
Since the *UsageWithPageCache* is calculated by `Total - Free` now, we should also change the calculation of the *UsageWithHotPageCache*.